mlrun/alerts/alert.py

```diff
@@ -350,8 +367,8 @@ def paginated_list_activations(
 
         :returns: A tuple containing the list of alert activations and an optional `page_token` for pagination.
         """
-        if from_last_update and self.updated is not None:
-            kwargs["since"] = self.updated_datetime
+        if from_last_update and self._updated is not None:
```

Comment:
`if from_last_update and self._updated` is better for the cases if `_updated` is an empty string to be on safe side.